app/src/main/java/com/kabouzeid/gramophone/adapter/song/PlayingQueueAdapter.java

```diff
@@ -22,17 +31,24 @@
 /**
  * @author Karim Abou Zeid (kabouzeid)
  */
-public class PlayingQueueAdapter extends SongAdapter implements DraggableItemAdapter<PlayingQueueAdapter.ViewHolder> {
+public class PlayingQueueAdapter extends SongAdapter
+        implements DraggableItemAdapter<PlayingQueueAdapter.ViewHolder>, SwipeableItemAdapter<PlayingQueueAdapter.ViewHolder> {
 
     private static final int HISTORY = 0;
     private static final int CURRENT = 1;
     private static final int UP_NEXT = 2;
 
+    public Song songToRemove;
+
     private int current;
 
+    private static AppCompatActivity activity;
```

Comment:
The parent class already holds a reference to `activity`.

Reply:
I missed that. Fixed.